clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -563,7 +563,7 @@ public static <T extends Throwable> void assertFutureThrows(
         String expectedMessage
     ) {
         T receivedException = assertFutureThrows(future, expectedCauseClassApiException);
-        assertEquals(expectedMessage, receivedException.getMessage());
+        assertTrue(receivedException.getMessage().contains(expectedMessage));
```

Comment:
Maybe we can remove duplicate `assertFutureThrows` and `assertFutureThrows` from this utils too?

Reply:
Do you mean `public static <T extends Throwable> T assertFutureThrows(Future<?> future, Class<T> exceptionCauseClass)` and `assertFutureThrows(Future<?> future, Class<T> expectedCauseClassApiException, String expectedMessage)` are duplicate?

Both overload has their references.
If I miss something, please tell me. Thanks! @chia7712

Reply:
Updated comment.

Reply:
Sorry for late response and unclear comment. I have opened https://issues.apache.org/jira/browse/KAFKA-18173 to address my comment